Documentation setting block size for overview --config GDAL_TIFF_OVR_BL… (#2736)
OSGeo/gdal: gdal/doc/source/programs/gdaladdo.rst

See the documentation of the :ref:`raster.gtiff` driver for further explanations on all those options.

> Setting blocksize in Geotiff overviewrs

Reviewer: ```suggestion
Setting blocksize in Geotiff overviews
```

---------------------------------------
--config GDAL_TIFF_OVR_BLOCKSIZE <size> 
example --config GDAL_TIFF_OVR_BLOCKSIZE 256

> default value is 128

Reviewer: ```suggestion
Default value is 128, or starting with GDAL 3.1, if creating overviews on a tiled GeoTIFF file, the tile size of the full resolution image.
```

> Note: without setting, the file can have level 0 with blocksize diffrent from overviews blocksize.(e.g. level 0 blocksize 256 ,levels 1-7 blocksize 128)

Reviewer: ```suggestion
Note: without this setting, the file can have the full resoultion image with a blocksize different from overviews blocksize.(e.g. full resolution image at blocksize 256, overviews at blocksize 128)
```
